src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/global.scss

```diff
@@ -55,6 +55,7 @@ h3 {
 button {
     border-radius: 0;
     padding: 0;
+    cursor: pointer;
```

Comment:
@FlorianBoe We are going with a component based css styling and avoid to have global styling for elements. So this should not be here and the `cursor: pointer` if it is missing at some component should be defined there in the component css. Adding button { cursor: pointer } here would make some unclickable disabled button now cursor pointer which can confuse users, only the component now which button state and which cursor make sense there. 

```suggestion
```